pacttesting/testing.go

```diff
@@ -139,21 +138,31 @@ func getVersion() (string, error) {
 	return version, nil
 }
 
-func setPathOnce() {
+func setBinPath() {
 	pathOnce.Do(func() {
-		topLevelDir, err := getTopLevelDir()
-		if err != nil {
-			panic(err)
+		if _, err := exec.LookPath("pact-mock-service"); err == nil {
+			return
+		}
+
+		binPath := os.Getenv("PACTTESTING_PATH")
+
+		if binPath == "" {
+			topLevelDir, err := getTopLevelDir()
+
+			if err != nil {
+				panic(err)
+			}
+
+			binPath = filepath.Join(topLevelDir, "pact/bin") + ":" + filepath.Join(topLevelDir, "tools/pact/bin")
 		}
-		pactPath = filepath.Join(topLevelDir, "pact/bin")
 
-		os.Setenv("PATH", pactPath+":"+os.Getenv("PATH"))
+		os.Setenv("PATH", os.Getenv("PATH")+":"+binPath)
```

Comment:
I think I'd avoid changing the `PATH` environment variable as there might be unintended side effects (as you're changing the variable for the whole environment, not only your executable). I'd probably go with specifying the full path instead.

Reply:
Unfortunately how it works right now we need to change the PATH since other code (need to check where) depends on the pact binaries to be on the path.

Reply:
Oh! That's unfortunate then.

Reply:
Also: we already set the path before so this is not new behavior